### ISSUE TITLE
WDS Trainer presentation (PPT)

### ISSUE THREAD

**Issue by DawnmarieDesJardins** [Microsoft/MCW-OSS-DevOps#17]
Please see slide 34.  Accessibility check says missing alt-test.  Alt-text is there but is references Jenkins and the screenshot showing is Visual Studio.  There is a Jenkins and Azure pipeline screenshots hidden underneath the current one.  If not needed, please remove.  Please verify that the alt-text matches what is on the slide.
